### PR TITLE
Follower Guard Overhaul

### PR DESCRIPTION
**Pull request by ValiantEsp**
Gives Follower Guards the ability to choose between an engineering, mining, or combat loadout. Rebrands them as 'scholars', and gives them traits to match. Now they have more reason to work as field workers.

Also hotfixes a few access things.

## About The Pull Request

Follower Guards are in a bit of an awkward spot now since Bighorn now has it's own defense force. This clears up some of the redundancy by giving them both a unique combat loadout (Hunting shotgun? Really?), and more they CAN do, giving them the ability to serve as field agents for the Followers... But no medical ability.

Also fixes Follower Admin access, and replaces the single weird bar door with a functional one.

## Why It's Good For The Game

As per the discussion in Suggestions, it's a much needed upgrade to Follower Guards. Renames them to Follower Scholars as well, to both indicate that they can do more now, add more Fallout Classic flair, but to avoid 'Field Researcher' or 'Field Scientist' for the sake of existing guards. Also yet another hotfix on access fixing. BECAUSE. I WILL FIX THOSE DOORS IF IT'S THE LAST THING I DO.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
tweak: Replaces Follower Guards with Follower Scholars, a broader role with more traits and loadouts.
fix: Fixes Follower Administrators not being able to access Bighorn Power Plant
fix: Fixes that one door in the bar.
tweak: Adds the 'Command' flag to the Follower Administrator, for future BH2.0 things, on the job screen. No real effect outside of manifest and job screen.
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/followers.dm [hunk 1]
```diff
@@ -345,24 +345,56 @@ Follower Volunteer
...
+	ADD_TRAIT(H, TRAIT_CYBERNETICIST_EXPERT, src)
+	ADD_TRAIT(H, TRAIT_CYBERNETICIST, src)
+
+/datum/outfit/loadout/engineer //Upgraded tools, RCD
+	name = "Followers Engineer"
+	belt = /obj/item/storage/belt/utility/full/engi
+	head =	/obj/item/clothing/head/hardhat
+	gloves =	/obj/item/clothing/gloves/color/yellow
+	backpack_contents = list(
+		/obj/item/construction/rcd = 1,
```
Comment (ProbablyCarl) [inline code/modules/jobs/job_types/followers.dm:366]: No, thanks. They'd be the only group to get something like this roundstart, no?
Check how Engineer is done for NCR.

File: code/modules/jobs/job_types/followers.dm [hunk 2]
```diff
@@ -345,24 +345,56 @@ Follower Volunteer
...
-/datum/outfit/loadout/guard_close
-	name = "Followers Close Protection Guard"
-	suit_store = /obj/item/gun/ballistic/shotgun/hunting
+/datum/outfit/loadout/combat //Guard, but actually good.
+	name = "Followers Crusader"
+	suit = /obj/item/clothing/suit/hooded/followerheavy
+	belt = /obj/item/storage/belt/military/army/military/
 	backpack_contents = list(
+		/obj/item/gun/energy/laser/wattz/recharger = 1,
+		/obj/item/gun/ballistic/shotgun/police = 1,
```
Comment (ProbablyCarl) [inline code/modules/jobs/job_types/followers.dm:387]: Could give them the recharger rifle, no?

Comment (ValiantEsp) [reply code/modules/jobs/job_types/followers.dm:387]: IS there a recharger rifle?

File: code/modules/jobs/job_types/followers.dm (hunk 1 above)
Comment (ValiantEsp) [reply code/modules/jobs/job_types/followers.dm:366]: RCDs themselves can be made in an autolathe, but, point! Replacing with metal and glass.

**Pull request merged**